tox.ini

```diff
@@ -29,3 +29,14 @@ commands =
         --non-interactive \
         --ignore-missing-imports \
         --package {posargs:"atomic_reactor"}
+
+[paths]
+source = atomic_reactor
+
+[coverage:report]
+skip_covered = true
+sort = Cover
+
+[pytest]
+addopts = -n=2 -ra --color=auto --html=__pytest_reports/atomic-reactor-unit-tests.html --self-contained-html
```

Comment:
Would `-n=auto` be better? (attempts to auto-detect number of cores and use that many processes)

Reply:
I would say it depends. From what I observed locally, `-n=auto` takes longer time to run the tests and the major incremental part is the preparation for processes and waiting the execution starts. Different environment might have different effect resulted by the `auto`. `-n=2` should have a good enough effect for local and CI as a default value.